src/baldr/datetime.cc

```diff
@@ -112,13 +113,12 @@ int timezone_diff(const uint64_t seconds,
   const auto origin = date::make_zoned(origin_tz, tp);
   const auto dest = date::make_zoned(dest_tz, tp);
 
-  auto duration = std::chrono::duration_cast<std::chrono::seconds>(origin.get_local_time() -
-                                                                   dest.get_local_time());
-  if (origin.get_info().offset < dest.get_info().offset) {
-    return std::abs(duration.count());
-  } else {
-    return -1 * std::abs(duration.count());
-  }
```

Comment:
under the hood `get_local_time` also calls `get_info`. `get_info` is quite a heavy call. it turns out you can use the info returned from it to do the timezone offset directly so just this small refactor ends up cutting our long time dependent routes response time in half.